tools/devtool

```diff
@@ -2075,7 +2076,7 @@ cmd_build_rootfs() {
         gcc -o  $rootfs_dir_ctr/readmem $resources_dir_ctr/readmem.c"
 
     img_file="$rootfs_dir_host/$ROOTFS_NAME"
-    dd if=/dev/zero "of=$img_file" bs=1M count=$SIZE
+    truncate -s "$SIZE" "$img_file"
```

Comment:
I like the usage of `truncate` because it parses the size part (i.e. MB, GB) for us automatically. However, I think the behavior is not the same as `dd` when the file already exists. Should we throw an error for this?

Reply:
Yes, let s thrown an error when the file already exists

Reply:
I did the changes. Take a look!